File: crossecore-java/src/java_/Annotation.java
package java_;
public interface Annotation 
 extends Expression
{
	
	public static com.crossecore.ocl.QuickSet<Annotation> allInstances_ = new com.crossecore.ocl.QuickSet<Annotation>(Annotation.class);
	
		
	public static com.crossecore.ocl.QuickSet<Annotation> allInstances(){
		
		com.crossecore.ocl.QuickSet<Annotation> result = new com.crossecore.ocl.QuickSet<Annotation>(Annotation.class);
		result.addAll(Annotation.allInstances_);
		
		
		return result;
	}
	
	//public static com.crossecore.ocl.QuickSet<Annotation> allInstances = new com.crossecore.ocl.QuickSet<Annotation>(Annotation.class);
	
	
	TypeAccess getType();
	void setType(TypeAccess value);
	com.crossecore.ocl.OrderedSet<AnnotationMemberValuePair> getValues();
	
}

File: crossecore-typescript/java_/Annotation.ts
import {OrderedSet} from "ecore/OrderedSet";
import {Set} from "ecore/Set";
import {AnnotationMemberValuePair} from "./AnnotationMemberValuePair";
import {Expression} from "./Expression";
import {TypeAccess} from "./TypeAccess";

export interface Annotation
extends Expression

{
	
	type:TypeAccess;
	values: OrderedSet<AnnotationMemberValuePair>;
	

}

